db/seeds.rb

```diff
@@ -206,6 +206,7 @@
 # other available definitions:
 controller_def = Definition.create!(content: "Communicates between the model (database) and views (user interface/website/browser)", word: controller)
 controller_def_2 = Definition.create!(content: "A controller is the waiter; between the model and the view. It determines what the user will be doing/seeing based on the request", word: controller)
+flash_hash_def = Definition.create!(content: "A way to pass information from one place to another ONLY ONCE (i.e. displaying on a login form the user entered the wrong password)", word: flash_hash, language: ruby, read_more: "url", meme: "url")
```

Comment:
Hi Waverly! 
Would you be okay with changing "from one place to another" to "from one controller action to another"? Just so folks don't think it's between e.g. model and controller :)

Reply:
Sure!